resources/ahk/TradeMacro.ahk

```diff
@@ -87,13 +87,6 @@ OpenSearchOnPoeTrade:
 	SuspendPOEItemScript = 1 ; This allows us to handle the clipboard change event
 	Send ^{sc02E}
 	Sleep 250
-	
-	TradeFunc_DoParseClipboard()
-	If (!Item.Name and TradeOpts.OpenUrlsOnEmptyItem) {
```

Comment:
Now that you mentioned the wiki, I looked at it and I think I see the problems, nothing related to my changes.
To achieve the behavior you are looking for, the clipboard would have to be cleared before sending the ctrl+c command.
Take a quick look at the 5 comments I added in the code below

```
OpenWiki:
	IfWinActive, Path of Exile ahk_class POEWindowClass 
	{
		Global TradeOpts, Item
		SuspendPOEItemScript = 1 ; This allows us to handle the clipboard change event
		
; the ctrl+c command won't reset or change the clipboard if no item is hovered
		Send ^{sc02E}
		Sleep 250
		
; parsing same old clipboard since no item was hovered
		TradeFunc_DoParseClipboard()

; now this part can only occur at script start when the clipboard is empty ( unless an item was parsed without a name for some reason )
		If (!Item.Name and TradeOpts.OpenUrlsOnEmptyItem) {
			TradeFunc_OpenUrlInBrowser("http://pathofexile.gamepedia.com/")
			
; this return would prevent ItemInfo to resume since SuspendPOEItemScript is never setted back to 0
			return
		}
		
; I'm not sure this can cause a problem but UrlAffix is never initialized
		If (Item.IsUnique or Item.IsGem or Item.IsDivinationCard or Item.IsCurrency) {
			UrlAffix := Item.Name
		} Else If (Item.IsFlask or Item.IsMap) {
			UrlAffix := Item.SubType
		} Else If (RegExMatch(Item.Name, "i)Sacrifice At") or RegExMatch(Item.Name, "i)Fragment of") or RegExMatch(Item.Name, "i)Mortal ") or RegExMatch(Item.Name, "i)Offering to ") or RegExMatch(Item.Name, "i)'s Key") or RegExMatch(Item.Name, "i)Breachstone")) {
			UrlAffix := Item.Name
		} Else {
			UrlAffix := Item.BaseType
		}
		
		If (StrLen(UrlAffix) > 0) {			
			UrlAffix := StrReplace(UrlAffix," ","_")
			WikiUrl := "http://pathofexile.gamepedia.com/" UrlAffix		
			TradeFunc_OpenUrlInBrowser(WikiUrl)	
		}
		
		SuspendPOEItemScript = 0 ; Allow Item info to handle clipboard change event
	}
return
```